src/bootstrap/server-bootstrapper.js

```diff
@@ -60,6 +62,20 @@ export async function createContainer(configValues = {}) {
   logger.info('Registering resource manager');
   container.registerInstance(ResourceManager, resourceManager);
 
+  logger.info('Loading theme files');
+  const globalVariablesFileUrl = new URL(path.resolve('./src/styles/global-variables.less'), import.meta.url);
```

Comment:
There's no need for using an URL as we only feed it to `fs.readFile(Sync)`, so I think it would be better to resolve the path to the global variables relative to the current module, basically the same way as we do for the resources above (line 20):

~~~
const globalVariablesFilePath = path.resolve(thisDir, '../styles/global-variables.less');
~~~

Reply:
Did it the same, thanks!